<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>1141</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 0, 38);
border-radius:35px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>0</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>151</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/icons8-close-window-48.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Start_pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>690</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #000;
padding: 5px 10px;
background-color: #fff;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-circled-play-48.png</normaloff>Images/icons8-circled-play-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Stop_pushButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/icons8-minimize-window-48.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/icons8-voice-48.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/icons8-no-audio-40.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/icons8-voice-48.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>710</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="command_pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>710</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
background-color: #000;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Commande</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-plus-40.png</normaloff>Images/icons8-plus-40.png</iconset>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="contact_pushButton_2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>710</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
background-color: #000;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Contacts</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-contact-48.png</normaloff>Images/icons8-contact-48.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="about_pushButton_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>710</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
background-color: #000;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-username-48.png</normaloff>Images/icons8-username-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="more_pushButton_4">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>710</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
background-color: #000;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Mores</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-info-48 (1).png</normaloff>Images/icons8-info-48 (1).png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>701</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
background-color: rgb(13, 0, 38);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gif1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>671</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/Gif/AI-visualization-design-unscreen.gif</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="volume_moins_push_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mute_push_button">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Volume_plus_pushButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>100</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hide_pushButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: transparent;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Facebook_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
	background-color: rgb(3, 155, 229);
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Facebook</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-facebook-48.png</normaloff>Images/icons8-facebook-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Youtube_pushButton_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
	background-color: rgb(234, 36, 52);
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Youtube</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-youtube-logo-48.png</normaloff>Images/icons8-youtube-logo-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Google_pushButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
background-color: #000;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Google</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-google-48.png</normaloff>Images/icons8-google-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Tikitok_pushButton_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #000;
padding: 5px 10px;
background-color: #fff;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>TikTok</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-tiktok-50.png</normaloff>Images/icons8-tiktok-50.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Instagramm_pushButton_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
color: #fff;
padding: 5px 10px;
background-color: #000;
border:none;
}
QPushButton:hover{
	background-color: rgb(203, 203, 203);
}</string>
   </property>
   <property name="text">
    <string>Insta</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-instagram-48.png</normaloff>Images/icons8-instagram-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>100</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:25;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>110</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Images/icons8-username-48.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_lineEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(0, 0, 0);
color:#fff;
border-radius:10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Utilisateur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_pushButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(0, 0, 0);
color:#fff;
border-radius:10px;
}
QPushButton:hover{
background-color: rgb(181, 181, 181);

}
</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/icons8-save-48.png</normaloff>Images/icons8-save-48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>Start_pushButton</zorder>
  <zorder>Stop_pushButton</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>command_pushButton</zorder>
  <zorder>contact_pushButton_2</zorder>
  <zorder>about_pushButton_3</zorder>
  <zorder>more_pushButton_4</zorder>
  <zorder>label_17</zorder>
  <zorder>gif1</zorder>
  <zorder>volume_moins_push_button</zorder>
  <zorder>mute_push_button</zorder>
  <zorder>Volume_plus_pushButton</zorder>
  <zorder>hide_pushButton</zorder>
  <zorder>Facebook_pushButton</zorder>
  <zorder>Youtube_pushButton_2</zorder>
  <zorder>Google_pushButton_3</zorder>
  <zorder>Tikitok_pushButton_4</zorder>
  <zorder>Instagramm_pushButton_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>user_lineEdit</zorder>
  <zorder>user_pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
